openwisp_notifications/static/openwisp-notifications/js/settings.js

```diff
@@ -0,0 +1,207 @@
+                    const row = $(
+                        '<tr>' +
+                        '<td>' + setting.type + '</td>' +
+                        '<td><input type="checkbox" class="checkbox email-checkbox" ' + (setting.email ? "checked" : "") + ' data-pk="' + setting.id + '" data-organization-id="' + setting.organization + '" data-type="email" /></td>' +
+                        '<td><input type="checkbox" class="checkbox web-checkbox" ' + (setting.web ? "checked" : "") + ' data-pk="' + setting.id + '" data-organization-id="' + setting.organization + '" data-type="web" /></td>' +
+                        '</tr>'
+                    );
+                    table.append(row);
+                });
+                orgContent.append(table);
+                updateMainCheckboxes(table);
+            } else {
+                orgContent.append('<div class="no-settings">No settings available for this organization</div>');
+            }
+            orgPanelsContainer.append(orgPanel);
+        });
+    }
+
+    function updateMainCheckboxes(table) {
+        table.find('.main-checkbox').each(function () {
+            const column = $(this).data('column');
+            const allChecked = table.find('.' + column + '-checkbox').length === table.find('.' + column + '-checkbox:checked').length;
+            $(this).prop('checked', allChecked);
+        });
+    }
+
+    function initializeEventListeners(userId) {
+        $(document).on('click', '.org-header', function () {
+            const toggle = $(this).find(".toggle");
+            toggle.text(toggle.text() === "▼" ? "▲" : "▼");
+            $(this).next(".org-content").toggleClass("active");
+        });
+
+        $(document).on('change', '.email-checkbox, .web-checkbox', function () {
+            if (isGlobalChange) {
+                return;
+            }
+            updateIndividualSetting(userId, $(this));
+            updateOrgLevelCheckboxes($(this).data('organization-id'));
+        });
+
+        $(document).on('change', '.main-checkbox', function () {
+            if (isGlobalChange) {
+                return;
+            }
+            updateOrganizationSetting(userId, $(this));
+            const table = $(this).closest('table');
+            table.find('.' + $(this).data('column') + '-checkbox').prop('checked', $(this).is(':checked'));
+            updateMainCheckboxes(table);
+        });
+    }
+
+    function updateIndividualSetting(userId, checkbox) {
+        const data = {};
+        data[checkbox.data('type')] = checkbox.is(':checked');
+        $.ajax({
+            type: 'PATCH',
+            url: '/api/v1/notifications/user/' + userId + '/user-setting/' + checkbox.data('pk') + '/',
+            headers: { 'X-CSRFToken': $('input[name="csrfmiddlewaretoken"]').val() },
+            contentType: "application/json",
+            data: JSON.stringify(data),
+            success: function () {
+                showToast('success', 'Settings updated successfully.');
```

Comment:
Please flag as translatable, look for the `gettext` function in this module or other modules.
```suggestion
                showToast('success', gettext('Settings updated successfully.'));
```
Applies to other user facing strings as well.

Reply:
Updated.